ignite/engine/engine.py

```diff
@@ -244,6 +244,10 @@ def print_epoch(engine):
             See :class:`~ignite.engine.Events` for more details.
 
         """
+        if isinstance(event_name, EventList):
+            for e in event_name:
+                self.add_event_handler(e, handler, *args, **kwargs)
+            return
```

Comment:
Shouldn't we still return something here ?
Otherwise, the docstring is incomplete.

Reply:
Indeed, we should return something, I didn't catch that. Taking a closer look, I think `add_event_handler` shouldn't be the way to add a list of events. A specific function should be defined.
@vfdev-5 thoughts ?

Reply:
`add_eventlist_handler` which returns a list of `RemovableEventHandler` ?

Reply:
See new commit. Skeptical about `add_eventlist_handler` naming... `add_event_handlers` ?